test/index.js

```diff
@@ -3,20 +3,18 @@ var memdown = require('memdown')
 var subdown = require('../leveldown')
 var levelup = require('levelup')
 var testCommon = require('./common')
-var testBuffer = new Buffer('this-is-test-data')
 
 require('abstract-leveldown/abstract/open-test').args(down, test, testCommon)
-require('abstract-leveldown/abstract/open-test').open(down, test, testCommon)
 require('abstract-leveldown/abstract/del-test').all(down, test, testCommon)
 require('abstract-leveldown/abstract/get-test').all(down, test, testCommon)
 require('abstract-leveldown/abstract/put-test').all(down, test, testCommon)
-require('abstract-leveldown/abstract/put-get-del-test').all(down, test, testCommon, testBuffer)
+require('abstract-leveldown/abstract/put-get-del-test').all(down, test, testCommon)
 require('abstract-leveldown/abstract/batch-test').all(down, test, testCommon)
 require('abstract-leveldown/abstract/chained-batch-test').all(down, test, testCommon)
 require('abstract-leveldown/abstract/close-test').close(down, test, testCommon)
 require('abstract-leveldown/abstract/iterator-test').all(down, test, testCommon)
-require('abstract-leveldown/abstract/ranges-test').all(down, test, testCommon)
+require('abstract-leveldown/abstract/iterator-range-test').all(down, test, testCommon)
```

Comment:
@finnp Did you have any help of the upgrade guide in `abstract-leveldown`?

Reply:
Yes, I had a look at it, that was quite helpful.

Reply:
That's awesome to hear! It paid off :)